### ISSUE TITLE
Chrome v52 breaks animation system (and maybe more?)

### ISSUE THREAD

**Issue by MaximilianWiedemann** [angular/angular#10245]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report
[ ] feature request
[ ] support request => Please do not submit support request here, instead see https://github.com/angular/angular/blob/master/CONTRIBUTING.md#question
```

**Current behavior**
In latest major version of Chrome (v52.0.2743.82), when loading an webpage in an angular 2 application that uses angular's animation system, it crashes the whole javascript part and therefore you can't interact with anything on the page.

The error that is being thrown:
```
EXCEPTION: Error in http://run.plnkr.co/L0iln61RH1WwNOpT/app/hero-list-enter-leave.component.ts class HeroListEnterLeaveComponent - inline template:3:10
platform-browser.umd.js:1900 EXCEPTION: Error in http://run.plnkr.co/L0iln61RH1WwNOpT/app/hero-list-enter-leave.component.ts class HeroListEnterLeaveComponent - inline template:3:10BrowserDomAdapter.logError @ platform-browser.umd.js:1900BrowserDomAdapter.logGroup @ platform-browser.umd.js:1910ExceptionHandler.call @ core.umd.js:4329(anonymous function) @ core.umd.js:9813schedulerFn @ core.umd.js:6699SafeSubscriber.__tryOrUnsub @ Subscriber.ts:240SafeSubscriber.next @ Subscriber.ts:192Subscriber._next @ Subscriber.ts:133Subscriber.next @ Subscriber.ts:93Subject._finalNext @ Subject.ts:154Subject._next @ Subject.ts:144Subject.next @ Subject.ts:90EventEmitter.emit @ core.umd.js:6687onError @ core.umd.js:9249onHandleError @ core.umd.js:9121ZoneDelegate.handleError @ zone.js@0.6.12?main=browser:327Zone.runGuarded @ zone.js@0.6.12?main=browser:233NgZoneImpl.runInnerGuarded @ core.umd.js:9133NgZone.runGuarded @ core.umd.js:9365outsideHandler @ platform-browser.umd.js:2722ZoneDelegate.invokeTask @ zone.js@0.6.12?main=browser:356Zone.runTask @ zone.js@0.6.12?main=browser:256ZoneTask.invoke @ zone.js@0.6.12?main=browser:423
platform-browser.umd.js:1900 ORIGINAL EXCEPTION: TypeError: Failed to execute 'animate' on 'Element': 'null' is not a valid value for easingBrowserDomAdapter.logError @ platform-browser.umd.js:1900ExceptionHandler.call @ core.umd.js:4338(anonymous function) @ core.umd.js:9813schedulerFn @ core.umd.js:6699SafeSubscriber.__tryOrUnsub @ Subscriber.ts:240SafeSubscriber.next @ Subscriber.ts:192Subscriber._next @ Subscriber.ts:133Subscriber.next @ Subscriber.ts:93Subject._finalNext @ Subject.ts:154Subject._next @ Subject.ts:144Subject.next @ Subject.ts:90EventEmitter.emit @ core.umd.js:6687onError @ core.umd.js:9249onHandleError @ core.umd.js:9121ZoneDelegate.handleError @ zone.js@0.6.12?main=browser:327Zone.runGuarded @ zone.js@0.6.12?main=browser:233NgZoneImpl.runInnerGuarded @ core.umd.js:9133NgZone.runGuarded @ core.umd.js:9365outsideHandler @ platform-browser.umd.js:2722ZoneDelegate.invokeTask @ zone.js@0.6.12?main=browser:356Zone.runTask @ zone.js@0.6.12?main=browser:256ZoneTask.invoke @ zone.js@0.6.12?main=browser:423
platform-browser.umd.js:1900 ORIGINAL STACKTRACE:BrowserDomAdapter.logError @ platform-browser.umd.js:1900ExceptionHandler.call @ core.umd.js:4341(anonymous function) @ core.umd.js:9813schedulerFn @ core.umd.js:6699SafeSubscriber.__tryOrUnsub @ Subscriber.ts:240SafeSubscriber.next @ Subscriber.ts:192Subscriber._next @ Subscriber.ts:133Subscriber.next @ Subscriber.ts:93Subject._finalNext @ Subject.ts:154Subject._next @ Subject.ts:144Subject.next @ Subject.ts:90EventEmitter.emit @ core.umd.js:6687onError @ core.umd.js:9249onHandleError @ core.umd.js:9121ZoneDelegate.handleError @ zone.js@0.6.12?main=browser:327Zone.runGuarded @ zone.js@0.6.12?main=browser:233NgZoneImpl.runInnerGuarded @ core.umd.js:9133NgZone.runGuarded @ core.umd.js:9365outsideHandler @ platform-browser.umd.js:2722ZoneDelegate.invokeTask @ zone.js@0.6.12?main=browser:356Zone.runTask @ zone.js@0.6.12?main=browser:256ZoneTask.invoke @ zone.js@0.6.12?main=browser:423
platform-browser.umd.js:1900 TypeError: Failed to execute 'animate' on 'Element': 'null' is not a valid value for easing
    at TypeError (native)
    at WebAnimationsDriver._triggerWebAnimation (https://npmcdn.com/@angular/platform-browser@2.0.0-rc.4//bundles/platform-browser.umd.js:1693:24)
    at WebAnimationsDriver.animate (https://npmcdn.com/@angular/platform-browser@2.0.0-rc.4//bundles/platform-browser.umd.js:1688:31)
    at DomRenderer.animate (https://npmcdn.com/@angular/platform-browser@2.0.0-rc.4//bundles/platform-browser.umd.js:2618:42)
    at DebugDomRenderer.animate (https://npmcdn.com/@angular/core@2.0.0-rc.4//bundles/core.umd.js:11743:35)
    at HeroListEnterLeaveComponent_flyInOut_0_factory (HeroListEnterLeaveComponent.template.js:20:127)
    at DebugAppView._View_HeroListEnterLeaveComponent1.detectChangesInternal (HeroListEnterLeaveComponent.template.js:139:5)
    at DebugAppView.AppView.detectChanges (https://npmcdn.com/@angular/core@2.0.0-rc.4//bundles/core.umd.js:12143:18)
    at DebugAppView.detectChanges (https://npmcdn.com/@angular/core@2.0.0-rc.4//bundles/core.umd.js:12247:48)
    at DebugAppView.AppView.detectContentChildrenChanges (https://npmcdn.com/@angular/core@2.0.0-rc.4//bundles/core.umd.js:12161:23)BrowserDomAdapter.logError @ platform-browser.umd.js:1900ExceptionHandler.call @ core.umd.js:4342(anonymous function) @ core.umd.js:9813schedulerFn @ core.umd.js:6699SafeSubscriber.__tryOrUnsub @ Subscriber.ts:240SafeSubscriber.next @ Subscriber.ts:192Subscriber._next @ Subscriber.ts:133Subscriber.next @ Subscriber.ts:93Subject._finalNext @ Subject.ts:154Subject._next @ Subject.ts:144Subject.next @ Subject.ts:90EventEmitter.emit @ core.umd.js:6687onError @ core.umd.js:9249onHandleError @ core.umd.js:9121ZoneDelegate.handleError @ zone.js@0.6.12?main=browser:327Zone.runGuarded @ zone.js@0.6.12?main=browser:233NgZoneImpl.runInnerGuarded @ core.umd.js:9133NgZone.runGuarded @ core.umd.js:9365outsideHandler @ platform-browser.umd.js:2722ZoneDelegate.invokeTask @ zone.js@0.6.12?main=browser:356Zone.runTask @ zone.js@0.6.12?main=browser:256ZoneTask.invoke @ zone.js@0.6.12?main=browser:423
platform-browser.umd.js:1900 ERROR CONTEXT:BrowserDomAdapter.logError @ platform-browser.umd.js:1900ExceptionHandler.call @ core.umd.js:4345(anonymous function) @ core.umd.js:9813schedulerFn @ core.umd.js:6699SafeSubscriber.__tryOrUnsub @ Subscriber.ts:240SafeSubscriber.next @ Subscriber.ts:192Subscriber._next @ Subscriber.ts:133Subscriber.next @ Subscriber.ts:93Subject._finalNext @ Subject.ts:154Subject._next @ Subject.ts:144Subject.next @ Subject.ts:90EventEmitter.emit @ core.umd.js:6687onError @ core.umd.js:9249onHandleError @ core.umd.js:9121ZoneDelegate.handleError @ zone.js@0.6.12?main=browser:327Zone.runGuarded @ zone.js@0.6.12?main=browser:233NgZoneImpl.runInnerGuarded @ core.umd.js:9133NgZone.runGuarded @ core.umd.js:9365outsideHandler @ platform-browser.umd.js:2722ZoneDelegate.invokeTask @ zone.js@0.6.12?main=browser:356Zone.runTask @ zone.js@0.6.12?main=browser:256ZoneTask.invoke @ zone.js@0.6.12?main=browser:423
platform-browser.umd.js:1900 DebugContext {_view: _View_HeroListEnterLeaveComponent1, _nodeIndex: 0, _tplRow: 3, _tplCol: 10}BrowserDomAdapter.logError @ platform-browser.umd.js:1900ExceptionHandler.call @ core.umd.js:4346(anonymous function) @ core.umd.js:9813schedulerFn @ core.umd.js:6699SafeSubscriber.__tryOrUnsub @ Subscriber.ts:240SafeSubscriber.next @ Subscriber.ts:192Subscriber._next @ Subscriber.ts:133Subscriber.next @ Subscriber.ts:93Subject._finalNext @ Subject.ts:154Subject._next @ Subject.ts:144Subject.next @ Subject.ts:90EventEmitter.emit @ core.umd.js:6687onError @ core.umd.js:9249onHandleError @ core.umd.js:9121ZoneDelegate.handleError @ zone.js@0.6.12?main=browser:327Zone.runGuarded @ zone.js@0.6.12?main=browser:233NgZoneImpl.runInnerGuarded @ core.umd.js:9133NgZone.runGuarded @ core.umd.js:9365outsideHandler @ platform-browser.umd.js:2722ZoneDelegate.invokeTask @ zone.js@0.6.12?main=browser:356Zone.runTask @ zone.js@0.6.12?main=browser:256ZoneTask.invoke @ zone.js@0.6.12?main=browser:423
zone.js@0.6.12?main=browser:260 Uncaught EXCEPTION: Error in http://run.plnkr.co/L0iln61RH1WwNOpT/app/hero-list-enter-leave.component.ts class HeroListEnterLeaveComponent - inline template:3:10

ORIGINAL EXCEPTION: TypeError: Failed to execute 'animate' on 'Element': 'null' is not a valid value for easing

ORIGINAL STACKTRACE:
TypeError: Failed to execute 'animate' on 'Element': 'null' is not a valid value for easing
    at TypeError (native)
    at WebAnimationsDriver._triggerWebAnimation (https://npmcdn.com/@angular/platform-browser@2.0.0-rc.4//bundles/platform-browser.umd.js:1693:24)
    at WebAnimationsDriver.animate (https://npmcdn.com/@angular/platform-browser@2.0.0-rc.4//bundles/platform-browser.umd.js:1688:31)
    at DomRenderer.animate (https://npmcdn.com/@angular/platform-browser@2.0.0-rc.4//bundles/platform-browser.umd.js:2618:42)
    at DebugDomRenderer.animate (https://npmcdn.com/@angular/core@2.0.0-rc.4//bundles/core.umd.js:11743:35)
    at HeroListEnterLeaveComponent_flyInOut_0_factory (HeroListEnterLeaveComponent.template.js:20:127)
    at DebugAppView._View_HeroListEnterLeaveComponent1.detectChangesInternal (HeroListEnterLeaveComponent.template.js:139:5)
    at DebugAppView.AppView.detectChanges (https://npmcdn.com/@angular/core@2.0.0-rc.4//bundles/core.umd.js:12143:18)
    at DebugAppView.detectChanges (https://npmcdn.com/@angular/core@2.0.0-rc.4//bundles/core.umd.js:12247:48)
    at DebugAppView.AppView.detectContentChildrenChanges (https://npmcdn.com/@angular/core@2.0.0-rc.4//bundles/core.umd.js:12161:23)

ERROR CONTEXT:
[object Object]Zone.runTask @ zone.js@0.6.12?main=browser:260ZoneTask.invoke @ zone.js@0.6.12?main=browser:423
```

In the previous major version of Chrome (v51.0.2683.0), latest Firefox (v47.0.1), latest Edge (v25.10586.0) and latest Internet Explorer (v11.494.10586.0), it works fine.

**Expected/desired behavior**
It should just work as intended.

**Reproduction of the problem**
In your official documentation (https://angular.io/docs/ts/latest/guide/animations.html), there is a live example (Plunker) which is affected as well: https://angular.io/resources/live-examples/animations/ts/plnkr.html

Go there and click in the preview on 'Add inactive hero' and it should show some errors about animation in your javascript console.

**What is the expected behavior?**
See above.

**What is the motivation / use case for changing the behavior?**
I hate bugs. I don't really know how to answer this question.

**Please tell us about your environment:**

* **Angular version:** 2.0.0-rc.4
 
* **Browser:** Chrome v52.0.2743.82
 
* **Language:** TypeScript 1.8.7 (transpiled to ES5)

**Comment by MaximilianWiedemann** [comment]
As a side question: Might this be unintended behavior/bug of Chrome or does angular have to fix something in their code?

**Comment by ericmartinezr** [comment]
See https://github.com/angular/angular/issues/9752

**Comment by matsko** [comment]
This will be fixed in RC5 with #9780
